src/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -102,7 +102,10 @@ public bool ExclusiveAddressUse
                     throw new InvalidOperationException(SR.net_tcplistener_mustbestopped);
                 }
 
-                _serverSocket.ExclusiveAddressUse = value;
+                if (_serverSocket != null)
```

Comment:
Can't this be reduced to this format:

```c#
_serverSocket?.ExclusiveAddressUse = value;
```

Same comment applies to other patterns in this PR.

Reply:
I don't think ?. can be used for assignment; this would be useful though.

Reply:
Sigh.  You're right. I was so used to see those patterns that I just expected it to work. :)

I guess I'll open a feature request for Roslyn team.